### PR TITLE
update build instructions and general cleanup

### PR DESCRIPTION
**Pull request by MattAtkinson**

### CODE REVIEW

File: creatorkit/Toolbox.md [hunk 1]
```diff
@@ -150,24 +150,36 @@ If your Clickers are powered on, they should be displayed on this page. You can
...
 <code>$ curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo</code>
 
 <code>$ chmod a+x ~/bin/repo</code>
 
+## Getting and building the code
 
-You should now be able to run the following in a directory of your choice.
+The following commands will sync to the master branch of the source repositories. To sync to a different branch append " -b <branch-name>" to the "repo init" command.
 
 <code>$ repo init -u  https://github.com/CreatorKit/manifest.git</code>
```
Comment (nikhil-zinjurde-imgtec) [inline creatorkit/Toolbox.md]: It is better to give the complete example command

repo init -u  https://github.com/CreatorKit/manifest.git -b <branch-name>

File: creatorkit/Toolbox.md [hunk 2]
```diff
@@ -150,24 +150,36 @@ If your Clickers are powered on, they should be displayed on this page. You can
...
 Initially you need to install dependencies required to build. run the following the terminal on your Ubuntu build machine
 
-<code>$ sudo apt-get -y install g++ gcc binutils bzip2 flex python perl make libncurses5-dev libssl-dev grep unzip gawk subversion zlib1g-dev build-essential git mercurial cmake python-nose python-lxml</code>
+<code>$ sudo apt-get -y install g++ gcc binutils bzip2 flex python perl make libncurses5-dev libncursesw5-dev libssl-dev grep unzip gawk subversion zlib1g-dev build-essential git mercurial cmake python-nose python-lxml</code>
+
+### Installing Contiki compiler
+
+In order to build Contiki code for Clicker boards you will need the XC32 compiler from Microchip. For 64-bit Ubuntu, 32-bit runtime libraries must be installed before the XC32 compiler can be run:
+
+<code>$ sudo apt-get install libc6:i386</code>
```
Comment (nikhil-zinjurde-imgtec) [inline creatorkit/Toolbox.md]: lib32ncurses5-dev may also be required

Comment (MattAtkinson) [inline creatorkit/Toolbox.md]: May be required, or is required?

**Comment by nikhil-zinjurde-imgtec** [comment]
Approved & verified

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

**Pull request merged**